components/host-dashboard/AddFundsModal.js

```diff
@@ -446,14 +460,14 @@ const AddFundsModal = ({ collective, ...props }) => {
                         <CollectivePickerAsync
                           inputId={field.id}
                           data-cy="add-funds-source"
-                          types={['USER', 'ORGANIZATION']}
-                          creatable
```

Comment:
@kewitz : Seems the ability to create a collective as a new source in the AddFundsModel is remove here. Is this intentional?